src/devices/wifi/nm-wifi-utils.h

```diff
@@ -26,6 +26,7 @@
 #include "nm-setting-wireless.h"
 #include "nm-setting-wireless-security.h"
 #include "nm-setting-8021x.h"
+#include "nm-iwd-manager.h"
```

Comment:
I think `nm-wifi-utils.h` must not include iwd related headers. If you need it for the enum, then move the definition of the enum as well.